backend/main.py

```
@@ -0,0 +1,312 @@
+from fastapi import FastAPI, HTTPException, Depends, status
```

Ruff (0.12.2)

`fastapi.Depends` imported but unused

Remove unused import

(F401)